app/rtc_s1_parallel.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+
+'''
+RTC Workflow
+'''
+
+import argparse
+import multiprocessing
+import os
+import subprocess
+import time
+import yaml
+
+from rtc.runconfig import RunConfig
+
+
+def get_rtc_s1_parser():
+    '''Initialize YamlArgparse class and parse CLI arguments for OPERA RTC.
+    Modified after copied from `rtc_s1.py`
+    '''
+    parser = argparse.ArgumentParser(description='',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('run_config_path',
+                        type=str,
+                        default=None,
```

Comment:
Maybe I'm not fully familiar with argparse, but setting `nargs=1` did not work. Removing that parameter did work in the way I intended.